README.md

```diff
@@ -157,3 +226,7 @@ This section applies to Solidity and JavaScript projects alike.
 [soliumrc]: ./.soliumrc.json
 [travisci]: https://travis-ci.org/
 [webpack]: https://webpack.js.org/
+[Assigning issues and pull requests to other GitHub users]: https://help.github.com/en/articles/assigning-issues-and-pull-requests-to-other-github-users
+[About pull request reviews]: https://help.github.com/en/articles/about-pull-request-reviews
+[The (written) unwritten guide to pull requests]: https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests
+[The Art of Humanizing Pull Requests (PR’s)]: https://hackernoon.com/the-art-of-humanizing-pull-requests-prs-b520588eb345
```

Comment:
I would sort the link list alphabetically, but it doesn't matter 😅

Reply:
@schemar: sure, seems sensible—will change.